### PR TITLE
Calculator

### PR DESCRIPTION
**Pull request by dina28**

### CODE REVIEW

File: Calculator/Calculator/ViewController.swift [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    @IBOutlet weak var resultTextField: UITextField!
+    var result = Float();
+    var currentNumber = Float();
+    var currentOp = String();
```
Comment (hasanmainul) [inline Calculator/Calculator/ViewController.swift]: use `var currentOperator: String!`

File: Calculator/Calculator/ViewController.swift [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view, typically from a nib.
+        currentOp = "="
+        resultTextField.text = ("\(result)");
```
Comment (hasanmainul) [inline Calculator/Calculator/ViewController.swift]: For casting, use `String(result)`

File: Calculator/Calculator/ViewController.swift [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+class ViewController: UIViewController {
+
+    @IBOutlet weak var resultTextField: UITextField!
+    var result = Float();
+    var currentNumber = Float();
+    var currentOp = String();
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view, typically from a nib.
```
Comment (hasanmainul) [inline Calculator/Calculator/ViewController.swift]: remove unnecessary remarks

File: Calculator/Calculator/ViewController.swift [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+        currentOp = "="
+        resultTextField.text = ("\(result)");
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
```
Comment (hasanmainul) [inline Calculator/Calculator/ViewController.swift]: remove unnecessary remarks

File: Calculator/Calculator/ViewController.swift [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    }
+
+    @IBAction func btnNumberInputAction(sender: UIButton) {
+        currentNumber = currentNumber * 10 + Float((Int)(sender.titleLabel!.text!)!)
+        resultTextField.text = ("\(currentNumber)")
```
Comment (hasanmainul) [inline Calculator/Calculator/ViewController.swift]: For casting, use `String(currentNumber)`

File: Calculator/Calculator/ViewController.swift [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+            result = result - currentNumber
+        case "*":
+            result = result * currentNumber
+        case "/":
+            result = result / currentNumber
+        default:
+            print("error")
+        }
+        currentNumber = 0
+        resultTextField.text = ("\(result)")
```
Comment (hasanmainul) [inline Calculator/Calculator/ViewController.swift]: For casting, use `String(result)`

File: Calculator/Calculator/ViewController.swift [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+            result = 0
+        }
+        currentOp = sender.titleLabel!.text! as String
+    }
+    
+    @IBAction func btnClearAction(sender: UIButton) {
+        currentOp = "="
+        currentNumber = 0
+        result = 0
+        resultTextField.text = ("\(result)")
```
Comment (hasanmainul) [inline Calculator/Calculator/ViewController.swift]: For casting, use `String(currentNumber)`

File: Calculator/Calculator/ViewController.swift (hunk 6 above)
Comment (dina28) [inline Calculator/Calculator/ViewController.swift]: beta fazlami koro?? 👎

File: Calculator/Calculator/ViewController.swift (hunk 7 above)
Comment (dina28) [inline Calculator/Calculator/ViewController.swift]: 👎

**Pull request merged**